.yarnrc.yml

```diff
@@ -1,5 +1,7 @@
-plugins:
-  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
-    spec: "@yarnpkg/plugin-interactive-tools"
+compressionLevel: mixed
 
-yarnPath: .yarn/releases/yarn-3.6.1.cjs
+enableGlobalCache: false
+
+yarnPath: .yarn/releases/yarn-4.0.0.cjs
+
+nodeLinker: pnp
```

Comment:
It seems to me the default nodeLinker in Yarn 4 is now `node-modules`.

Reply:
It's not, maybe you set it globally in `~/.yarnrc.yml`?

Reply:
Indeed, now I wonder why previous Yarn version generates pnp builds.🤔